pkg/util/validation/limits.go

```diff
@@ -78,6 +78,7 @@ type Limits struct {
 	MaxQueryParallelism          int            `yaml:"max_query_parallelism" json:"max_query_parallelism"`
 	MaxCacheFreshness            model.Duration `yaml:"max_cache_freshness" json:"max_cache_freshness"`
 	MaxQueriersPerTenant         int            `yaml:"max_queriers_per_tenant" json:"max_queriers_per_tenant"`
+	QueryVerticalShardSize       int            `yaml:"query_vertical_shard_size" json:"query_vertical_shard_size" doc:"hidden"`
```

Comment:
Hidden makes it disappear from https://cortexmetrics.io/docs/configuration/configuration-file/#limits_config

Users have no way to discover this exists